cmslite_metadata.view.lkml

```diff
@@ -12,4 +12,9 @@ view: cmslite_metadata {
   dimension: title {}
   dimension: hr_url {}
   dimension: language {}
+  dimension: page_language {
+    type: string
+    sql: ${TABLE}.language_name ;;
+    description: "The language specified in the page settings"
```

Comment:
Can you make this "The language specified in the CMS Lite page settings tab"?

Reply:
Ive updated the description. Language code was added by the developer to the script, but it wasn't requested by us. I added a dimension for it as per your suggestion however.